runtime/gc_structs/ConstantPoolObjectSlotIterator.cpp

```diff
@@ -73,6 +73,13 @@ GC_ConstantPoolObjectSlotIterator::nextSlot()
 		if(slotType == J9CPTYPE_METHODHANDLE) {
 			return &(((J9RAMMethodHandleRef *) slotPtr)->methodHandle);
 		}
+		if(slotType == J9CPTYPE_CONSTANT_DYNAMIC) {
```

Comment:
Would you mind to replace all if's to the switch? I believe it will work faster and look cleaner